Problems/ConstantDensityStar/constant_density_star_mgpc_newton_petsc.c

```diff
@@ -259,117 +260,140 @@ problem_init
   }
 
   
-  /* d4est_output_energy_norm_fit_t* fit = d4est_output_new_energy_norm_fit(d4est_amr->num_of_amr_steps + 1); */
+  /* d4est_norms_energy_norm_fit_t* fit = d4est_norms_new_energy_norm_fit(d4est_amr->num_of_amr_steps + 1); */
   
   /*   d4est_ip_energy_norm_data_t ip_norm_data; */
   /*   ip_norm_data.u_penalty_fcn = sipg_params->sipg_penalty_fcn; */
   /*   ip_norm_data.sipg_flux_h = sipg_params->sipg_flux_h; */
   /*   ip_norm_data.penalty_prefactor = sipg_params->sipg_penalty_prefactor; */
 
-    d4est_output_norms_using_analytic_solution
```

Comment:
Why did you call the output function here before the loop already? I think I shuffled the code in the loop below in a way such that both the initial and final state are output,

Reply:
Probably to give the error associated with the initial guess, if not for this then I have no idea :). If your version gives the initial guess error then that is fine.